### PR TITLE
Security policy notes changes

### PR DESCRIPTION
**Pull request by timlegge**

### CODE REVIEW

File: docs/security-policy-for-authors.md [hunk 1]
```diff
@@ -217,7 +215,7 @@ Note that while security issues can occur due to unsafe defaults or
 the misuse of software libraries, it is still worth adding a section
 that highlights usage issues.  For example,
 
-> The "foo" method only validates that filenames passed to it exist.
+> The "foo" method only validates that filenames passed to it exists.
```
Comment (robrwo) [inline docs/security-policy-for-authors.md:218]: filenames ... exist. This isn't a typo.

**Review by robrwo — CHANGES_REQUESTED** [review]
Thanks for catching these typos. But there's one case that isn't a typo. Please fix and I will merge that.

I also may amend the note about contacting CPANSec if the reporter hasn't heard from the maintainer. It should be something like "Please send another message to the maintainer as a reminder and copy CPANSec (if CPANSec has not already been copied)."

**Comment by timlegge** [comment]
You're right. Not sure why. I changed that.

File: docs/security-policy-for-authors.md (hunk 1 above)
Comment (robrwo) [reply docs/security-policy-for-authors.md:218]: I'll merge and fix this separately

**Comment by timlegge** [comment]
Perfect, thanks. Meant to message you this morning that I probably wouldn't
get to it till till this evening

**Pull request merged**